lib/Data/Validate/Sanctions.csv

```diff
@@ -2330,52 +2009,41 @@ RZOOKIHANNA
 SAAVEDRAARCERODRIGOEUGENIO
 SAAVEDRARESTREPOJESUSMARIA
 SABAGHCAJELIROMEZJOSE
-SABETJAVADKARIMI
 SABOGALZULUAGAORLANDO
 SABORICISNEROSRAUL
 SABOUNIIMADABDULGHANI
-SADATAHMAD
 SADIKOVOLIMZHONADKHAMOVICH
 SAEEDMUHAMMAD
 SAEEDTALHA
 SAENZVARGASGUILLERMOLEON
 SAEZDEEGUILAZMURGUIONDOCARLOS
 SAFARADEL
 SAFAVIYAHYARAHIM
-SAFDARISEYEDJABER
 SAHINPASICSENAD
 SAHIRONRADULAN
 SAIDMOHAMED
 SAIFULLAHQARI
 SAIIDSHAYKH
 SAINOVICNIKOLA
 SAKABUYAMORRIS
-SAKUPWANYASTANLEY
 SALAHMUHAMMAD
-SALAMEHSALEM
 SALAZARCARDENASCARLOSMARIO
 SALAZARHERNANDEZMARIAALEJANDRINA
-SALAZARLUGONELSON
 SALAZARRAMIREZJESUSALFREDO
 SALAZARUMANAJOSEADAN
 SALCEDORAMIREZNHORACLEMENCIA
 SALEHALIMOHAMAD
 SALEHALSAADINASSIMBENMOHAMEDALCHERIFBENMOHAMED
 SALEHBACHIR
 SALEHDRABDULLATIF
-SALEHJAMAL
 SALEHKASSEMMOHAMAD
-SALEHMARIAMMAHMOUD
-SALEHMOHAMMEDABDELHALIMHEMAIDA
 SALEHNEDAL
 SALGUEIRONEVAREZNOEL
-SALHABAZZAM
 SALIHABDALMUNIMAHMAD
 SALIJRJAINALANTEL
 SALIMABDALRAHMANOULDMUHAMMADALHUSAYNOULDMUHAMMAD
 SALIMIHOSEIN
 SALINASPEREZOVIDIO
-SALMANMAALIM
```

Comment:
@fayland I think there was a problem with the download: when I run the same script, I don't see some of the removals, such as this one.

Maalim Salman is still listed in my copy of the CSV and shows up in https://sanctionssearch.ofac.treas.gov/ - could you check whether the two files were successfully retrieved?